utils/hare-reportbug

```diff
@@ -79,6 +79,9 @@ exec 2> >(tee _reportbug.stderr >&2)
 sudo journalctl --no-pager --full --utc --output short-precise \
      > syslog.txt || true
 
+sudo journalctl --no-pager --full --utc --output=json --unit=pacemaker.service \
+     > syslog.json || true
```

Comment:
maybe `s/syslog.json/syslog-pacemaker.json/`?

Reply:
Right. Fixed. Let me squash that fixup